Description: Correct pod NAME issue in LedgerSMB/File/Order.pm
 Has the module name but not the dash plus short description in the NAME entry.
Bug: http://sourceforge.net/tracker/?func=detail&aid=3472598&group_id=175965&atid=1005191
Author: Robert James Clay <jame@rocasa.us>

--- a/LedgerSMB/File/Order.pm
+++ b/LedgerSMB/File/Order.pm
@@ -2,7 +2,7 @@
 
 =head1 NAME
 
-LedgerSMB::File::Order
+LedgerSMB::File::Order - Manages attachments to orders.
 
 =head1 SYNPSIS
 
